### PR TITLE
Inferred Mode

### PR DESCRIPTION
**Pull request by cbizon**
Recording the decision that inferred mode queries must include lookup results as a subset.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,11 @@ This README documents the current strawman architecture.  Changes must be made v
...
    1. Making predicates more specific, e.g. responding to a query for “affects expression of” with an edge with predicate “increases expression of”.  In the response, the more specific edge must occur in the knowledge_graph portion of the response, and in individual results, that more specific edge will be bound to the less specific query edge.  Query Graph and Knowledge Graph edges need not match in either predicate or direction to be bound in an answer.
    1. Inverting symmetric predicates, e.g. if the KP contains information that A and B are correlated, then it should respond with that information whether the query is asked in the form A-[correlated_with]->B or B-[correlated_with]->A.
+1. Query Modes:
+   1. As described in the TRAPI specification, edges may be queried in either "lookup" or "inferred" mode.
+   1. KPs and ARAs must respond to lookup queries by treating the query as an exact database match
+   1. ARAs must respond to inferred mode one-hops with relevant results beyond an exact database match; KPs may also provide this capability
```
Comment (edeutsch) [inline README.md:63]: Perhaps pedantic, but what should be the behavior of an ARA that does not support inferred mode for a particular query? It is required by this to do something it cannot do. Should it error out and do nothing? Or return exact database matches and no inferred results? Which seems like a MAY for ARAs then too?

Comment (gglusman) [reply README.md:63]: Agreed that 'may' sounds much more appropriate.
Alternatively, ARAs could invoke ChatGPT and ask it to make something up so they can respond. ;)

**Review by vdancik — APPROVED** [review]

**Review by bill-baumgartner — APPROVED** [review]

**Review by marcdubybroad — APPROVED** [review]

**Review by andrewsu — APPROVED** [review]

**Review by newgene — APPROVED** [review]

**Review by edeutsch — APPROVED** [review]

**Review by uhbrar — APPROVED** [review]

**Review by CaseyTa — APPROVED** [review]

**Comment by dkoslicki** [comment]
@cbizon I had to miss the Architecture meeting: was there a discussion about how # of results might result in either creative mode results or look-up results getting truncated and thus not being present? 
And/or a discussion about how the UI might be able to identify which results are which? You mentioned above that this might be the case in TRAPI 1.4, but don't know if this was confirmed.

**Comment by andrewsu** [comment]
We have https://github.com/NCATSTranslator/Feedback/issues/95 for the July 30 milestone, and that depends on resolution here.  In an effort to move that along...

@edeutsch does TRAPI 1.4 have the ability to indicate creative vs lookup results? 

@dkoslicki If the answer above is "yes", then I assume you'd be fine with approving that and implementing.  If the answer is "no", then would you oppose this PR as the intended behavior for Sept?

**Comment by cbizon** [comment]
Based on the discussion at architecture

* The inferred/lookup distinction is clear to the UI team from the way that the TRAPI is structured
* ARAs can control their own filtering to make sure that they return however many creative edges they like along the lookup edges.

Merging...

**Pull request merged**